examples/python/geometry/octree.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
Please clear all outputs for cells before checking-in the notebook. The CI will run the notebook automatically.